src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx

```diff
@@ -47,6 +49,6 @@ export default function CancelAppointmentFailedModal({
           </>
         )}
       </p>
-    </Modal>
+    </VaModal>
   );
 }
```

Comment:
You can add propTypes to get rid of the linting warnings just like you would find in `CancelAppointmentConfirmationModal.jsx`